### DISCUSSION TITLE
How to configure a previous prompt? (transient prompt)

### DISCUSSION THREAD

**Discussion by felipevcc** [starship/starship#4051] [Q&A] [answered]
![Screenshot_2022-06-09-45_146x106](https://user-images.githubusercontent.com/95534180/172942261-e5c42ee2-ca21-459e-898a-ff8fe7218136.png)
Does anyone know how to set a different prompt than normal when a command has already been executed?
As shown in the image, when something has already been executed, only an icon is shown in the previous prompt

Something like the "transient prompt" from p10k
Or is this not possible?

**Comment by andytom** [comment, marked as answer]
This is not possible at the moment but there is an open issue, #888, you can follow for updates.
